kodeverk/src/main/java/no/nav/ung/kodeverk/produksjonsstyring/OmrådeTema.java

```diff
@@ -0,0 +1,102 @@
+package no.nav.ung.kodeverk.produksjonsstyring;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonFormat.Shape;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import no.nav.ung.kodeverk.api.Kodeverdi;
+
+@JsonFormat(shape = Shape.OBJECT)
+@JsonAutoDetect(getterVisibility = Visibility.NONE, setterVisibility = Visibility.NONE, fieldVisibility = Visibility.ANY)
+public enum OmrådeTema implements Kodeverdi {
```

Comment:
Skulle vi flyttet kodeverk som er veldig statisk og felles på tvers, for eksempel denne og  FagsakYtelseType til k9-felles eller noe lignende sted for å slippe å ha flere kopier?

Tenker ikke at vi skal gjøre det som del av denne PR-en

Reply:
Det hadde kanskje ikkje vore så dumt.